spring-cloud-config-client/src/main/java/org/springframework/cloud/config/client/ConfigClientProperties.java

```diff
@@ -117,6 +117,47 @@ public class ConfigClientProperties {
 	 */
 	private String password;
 
+	/**
```

Comment:
Might be cleaner to place these properties in a subclass called oauth2 so we would use `spring.cloud.config.oauth2.*`

Reply:
Done. See ConfigClientOauth2Properties